ui/package.json

```diff
@@ -125,7 +125,6 @@
     "ember-cli": "~4.12.1",
     "ember-cli-autoprefixer": "^0.8.1",
     "ember-cli-babel": "^7.26.11",
-    "ember-cli-clipboard": "0.16.0",
```

Comment:
Removed `ember-cli-clipboard` 0.16.0 because we no longer have structure copy button which relied on this. HDS copy button is dependent on `ember-cli-clipboard` 1.0.0, which was added when I upgraded HDS to 2.9.0 in first subtask of copy button replacement